web_client/analysis-modules/igpse-feature-selection.jsx

```diff
@@ -12,6 +12,7 @@ const main = (igpse) => (
   .then(() => igpse.renderElements('featureSelection'))
   .then(() => igpse.enablePage('featureSelection'))
   .then(() => igpse.switchPage('featureSelection'))
+  .then(() => igpse.setBusy(false))
```

Comment:
The whole point of the IGPSEWorkflow class was to basically stash complex logic and the like behind various shorthands that made coordinating the higher level steps easier.  Making a `setBusy` shorthand to go along with the others just made sense, imo.